.goreleaser.yml

```diff
@@ -5,16 +5,17 @@ builds:
 - main: ./cli
   env:
   - CGO_ENABLED=0
-archive:
-  replacements:
-    darwin: Darwin
-    linux: Linux
-    windows: Windows
-    386: i386
-    amd64: x86_64
-  format_overrides:
-    - goos: windows
-      format: zip
+archives:
+  - id: archive
+    replacements:
+      darwin: Darwin
+      linux: Linux
+      windows: Windows
+      386: i386
+      amd64: x86_64
+    format_overrides:
+      - goos: windows
+        format: zip
```

Comment:
It looks like this duplicates PR#56. Not a huge deal, but as we merge to master, I'd like to keep that tree clean, and the commits clearly labeled. Make sure you have the right bases in each PR.

Reply:
yep, I needed the other one merged here so that I could test the deploys for building docker images. Once the other PR is merged I can clean this up.

Reply:
cleaned up with the latest merge to master 😄

Reply:
:+1: